.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--- Please provide a general summary of your changes in the Title above -->
+
+# PR Description
+
+<!--- Describe the changes your PR makes in detail -->
+
+References issue: # (ISSUE)
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Tests and Coverage
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how your change affects other areas of the code, etc. -->
```

Comment:
I think it's a really good idea to ask something like this - so great you got this introduced 👏 I think we could dial down on the 'level of details' we ask them to include. In a way, do we care about their testing environment and which tests they run? My - potentially gullible - view on this is: if you covered your code with tests, the tests look reasonable (we'll have to review the code anyway no matter what they write in the PR) and they all pass, then we're golden. Do we need to ask for additional info on top of that? Just thinking that this may defer some devs from ever submitting a PR.

Reply:
Just to clarify - I meant that we're already asking 'have all tests passed?' question in the last section of the PR